.travis.yml

```diff
@@ -4,7 +4,7 @@ addons:
     secure: M4uBGUMbanDtCBrNktJhEwNSLL0UsOV3IoINqW3GRupX7Mr13MBZzsORm4HVbdbPxQ6Bf2LhyJ/fabQRsOoJrCokgWwfKkigztQcMFeLNao9liBRA6kuQOh5/rdfoULpu96GvlYzB4ddgSBSSGQW3DesaKC/BpTXnvQ4OnXo5e4=
 env:
   global:
-  - SAUCE_USERNAME=snay
+  - SAUCE_USERNAME=jh3141
```

Comment:
I'm not sure why you are changing the testing information. We obviously don't want to do this